File: iOS/iOS-MVVM/iOS-MVVM/Models/Cast.swift
//
//  Cast.swift
//  iOS-MVVM
//
//  Created by Guillian on 2019-04-16.
//  Copyright © 2019 AffinityClick. All rights reserved.
//

import Foundation

struct Cast: Decodable {
    let castId: Int
    let character: String
    let creditId: String
    let gender: Int
    let name: String
    let id: Int
    let order: Int
    let profilePath: String
}

File: android/MVVM/app/src/main/java/com/affinityclick/mvvm/network/models/Cast.kt
package com.affinityclick.mvvm.network.models

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize
import kotlinx.parcelize.RawValue

@Parcelize
data class Cast(
  @SerializedName("cast_id") @Expose var castId: Int?,
  @SerializedName("character") @Expose var character: String?,
  @SerializedName("credit_id") @Expose var creditId: String?,
  @SerializedName("gender") @Expose var gender: Int?,
  @SerializedName("id") @Expose var id: Int?,
  @SerializedName("name") @Expose var name: String?,
  @SerializedName("order") @Expose var order: Int?,
  @SerializedName("profile_path") @Expose
  var profilePath: @RawValue Any? = null
) : Parcelable;